packages/vaadin-avatar/src/vaadin-avatar-group-overlay.js

```diff
@@ -0,0 +1,6 @@
+/**
+ * @license
+ * Copyright (c) 2021 Vaadin Ltd.
+ * This program is available under Apache License Version 2.0, available at https://vaadin.com/license/
+ */
+export * from '@vaadin/avatar-group/src/vaadin-avatar-group-overlay.js';
```

Comment:
We only need to re-export `vaadin-avatar` and `vaadin-avatar-group` so this file can be removed.